src/neurostatslib/utils.py

```diff
@@ -340,9 +339,10 @@ def plot_spike_raster(
 
 
 def row_wise_kron(A: jnp.array, C: jnp.array, jit=False, transpose=True) -> jnp.array:
-    """
-    Compute the row-wise Kronecker product between two matrices using JAX. See [1]
-    for more details on the Kronecker product.
+    """Compute the row-wise Kronecker product.
+
+    Compute the row-wise Kronecker product between two matrices using JAX.
+    See [1] for more details on the Kronecker product.
```

Comment:
These footnotes aren't rendering correctly, and I can't quickly seem to figure them out. The style you're using here looks like the sphinx syntax, but I tried switching to the [footnotes](https://squidfunk.github.io/mkdocs-material/reference/footnotes/) as described in mkdocs material, and it doesn't work (even with the footnotes plugin) -- any thoughts?

Reply:
I played around with the footnotes, it seems that if i add the footnot within block, they are rendered, e.g. this works but it is weird:

```python
     """Linearly-spaced raised cosine basis functions used by Pillow et al.[^1]

    These are "cosine bumps" that uniformly tile the space.

    [^1]:
        Pillow, J. W., Paninski, L., Uzzel, V. J., Simoncelli, E. P., & J.,
        C. E. (2005). Prediction and decoding of retinal ganglion cell responses
        with a probabilistic spiking model. Journal of Neuroscience, 25(47),
        11003–11013. http://dx.doi.org/10.1523/jneurosci.3305-05.2005

    Parameters
    ----------
    n_basis_funcs
        Number of basis functions.
    """ 
```

While this doesn't:
```
    """Linearly-spaced raised cosine basis functions used by Pillow et al.[^1]

    These are "cosine bumps" that uniformly tile the space.


    Parameters
    ----------
    n_basis_funcs
        Number of basis functions.

    References
    ----------
    [^1]:
        Pillow, J. W., Paninski, L., Uzzel, V. J., Simoncelli, E. P., & J.,
        C. E. (2005). Prediction and decoding of retinal ganglion cell responses
        with a probabilistic spiking model. Journal of Neuroscience, 25(47),
        11003–11013. http://dx.doi.org/10.1523/jneurosci.3305-05.2005

    """
```